Rethinking_2/End_of_chapter_problems/Chapter 3 - SAMPLING THE IMAGINARY.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>Do not use <code>start = pm.find_MAP()</code>, in general is not a good idea. For most simple model you will not need to do burnin. Also notice that PyMC3 by defaults runs 1000 steps of tuning and those steps are discarded.</p><p> </p><p>Use ArviZ function explicetiely, like az.plot_trace, az.plot_forest (notice that the correct argument name is <code>var_names</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>